NitroxServer/ServerAutoFaqRegistrar.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Autofac;
+using NitroxModel.Core;
+using System.Reflection;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.Communication;
+using NitroxServer.Serialization.World;
+using NitroxServer.GameLogic.Entities;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.Communication.Packets;
+using NitroxServer.ConfigParser;
+using NitroxServer.ConsoleCommands.Processor;
+using NitroxServer.Communication.Packets.Processors;
+
+namespace NitroxServer
+{
+    public class ServerAutoFaqRegistrar : IAutoFacRegistrar
+    {
+        private static World world;
+        private static ServerConfig config;
+
+        public ServerAutoFaqRegistrar(World world, ServerConfig config)
```

Comment:
Can the ServerConfig dependency be registered without passing an instance to this constructor? 

Ex: `containerBuilder.RegisterType<ServerConfig>().SingleInstance();`
Or: `containerBuilder.Register(c => ServerConfig.LoadFromFile("filepath")).SingleInstance()`

Ideally, `World` would also be registered like this but it's a big type and probably can be fixed by a good refactor of the World saving process. In short, It's fine for now.

Reply:
actually yeah its possible to do this now that I'm looking at it...